src/webview/__tests__/generateInboundEventEditSequence-test.js

```diff
@@ -224,7 +238,17 @@ const baseBackgroundData = {
  * and `messageListElementHtml`.
  */
 describe('messages -> piece descriptors -> content HTML is stable/sensible', () => {
-  const check = ({ backgroundData = baseBackgroundData, narrow, messages }) => {
+  const check = ({
+    state = baseState,
+    globalState = assumeSecretlyGlobalState(state),
```

Comment:
I'd rather not proliferate calls to `assumeSecretlyGlobalState` like this -- each one of them is effectively a TODO that will need to be resolved in order to complete #5006.

In this case (and that of the other `check` function below), I think what's really happening semantically is that it wants a copy of the *account-independent* part of the state. That is, all the parts of the global state that *aren't* any account's state.

We don't yet have a type for expressing that directly. Perhaps I should add one. But I think two good ways to express what these test functions want in the meantime would be:

Use a global state from example data:
```js
    globalState = eg.plusReduxState,
```
It's fine that that may not agree with the per-account state it gets, because we're really only looking at the account-independent parts of this -- for everything account-specific, we rely on that per-account state.

Or pass the two pieces we actually use, separately:
```js
  globalSettings = getGlobalSettings(eg.plusReduxState),
  debug = getDebug(eg.plusReduxState),
```

No existing call site gets any more verbose, because they don't actually pass these anyway. Future call sites mostly won't either, because they'll be passing only the pieces that those respective tests care about.

Reply:
Yeah, seems sensible. My next revision uses the "pass the two pieces we actually use" approach.